### DISCUSSION TITLE
Limit of 128 characters

### DISCUSSION THREAD

**Discussion by webwude** [mrikirill/SynologyDDNSCloudflareMultidomain#16] [Q&A]
Hi.

When I enter my domains and subdomains (I have quite a few of them) I realized that there is a limit of 128 characters. Is there any way to extend it?

Thx for an answer,
best regards
M.

**Comment by mrikirill** [comment]
Seems to be the same issue here -> https://github.com/mrikirill/SynologyDDNSCloudflareMultidomain/issues/17

**Comment by kennethpiu** [comment]
I also want to know since i have bulky DDNS with cloudflare need to be update and creating another script only brings "DDNS function error please contact synology technical support":(

**Reply by kennethpiu** [reply]
Update: it works!, just cloudflare no longer support DDNS update for the LTD those provided by [Freenom] (https://www.freenom.com/), but panel update ip fine with these domains. e.g. domain.ga .domain.tk etc.,

**Comment by mrikirill** [comment]
I've added info about 128 symbols limit into readme
